src/m365/tenant/commands/people/people-profilecardproperty-add.ts

```diff
@@ -0,0 +1,169 @@
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import request from '../../../../request.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import commands from '../../commands.js';
+import { profileCardPropertyNames } from './profileCardProperties.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  name: string;
+  displayName?: string;
+}
+
+class TenantPeopleProfileCardPropertyAddCommand extends GraphCommand {
+  public get name(): string {
+    return commands.PEOPLE_PROFILECARDPROPERTY_ADD;
+  }
+
+  public get description(): string {
+    return 'Adds an additional attribute to the profile card properties';
+  }
+
+  constructor() {
+    super();
+
+    this.#initTelemetry();
+    this.#initOptions();
+    this.#initValidators();
+  }
+
+  #initTelemetry(): void {
+    this.telemetry.push((args: CommandArgs) => {
+      Object.assign(this.telemetryProperties, {
+        name: args.options.name,
```

Comment:
This isn't required to be assigned to telemetry as `name` is a required option

Reply:
Hi @Jwaegebaert, it is not required, but as it is not PII, it's interesting information nonetheless. That's the idea of telemetry, that we log interesting stuff, usage and non-PII sensitive material. Which is why I included it.

Reply:
If we start adding this we'll be changing what we track in telemetry. We don't track any required options even if they aren't non-PII sensitive. ref: https://pnp.github.io/cli-microsoft365/about/telemetry#fn-1

Reply:
We would now be able to see what properties people generally add to profile cards. Which could drive our own ideas and goals. Anyway, maybe you can include it again before actually releasing the beta.

Reply:
Sure, we could consider adding that, but it would mean we'd need to take a closer look at our overall telemetry setup. Right now, we're saying we don't monitor required options, so even though it might be intriguing to track, it's not something we should include here.

Reply:
Do you mean `authType` within the `m365 login` command? Because in `m365 login` it's optional.

I would suggest, if you're really keen on implementing this, that we track it in a separate issue so we can discuss it more globally and then we can continue here.

Reply:
It's fine, we'll discuss it another time. It's not that important. I'm just saying we don't necessarily only track optional options.